servicedirectory/quickstart.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import servicedirectory_v1beta1
+
+
+def list_namespaces(project_id, location_id):
+  """Lists all namespaces in the given location."""
+
+  client = servicedirectory_v1beta1.RegistrationServiceClient()
+
+  response = client.list_namespaces(
+      parent='projects/{0}/locations/{1}'.format(project_id, location_id))
+
+  print('Listed namespaces in {0}.'.format(location_id))
```

Comment:
Throughout the code, consider using [f-string](https://www.python.org/dev/peps/pep-0498/) unless you want to support Python 3.5 or lower.